### PR TITLE
feat(Value.ArraySelection): add `variant` prop to allow for list layout

### PR DESCRIPTION
**Pull request by joakbjerk**
Sorry, need to fix some last minute changes 🙏

- [x] Remove unstyled lists
- [x] add ListType

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Value/ArraySelection/ArraySelectionDocs.ts [hunk 1]
```diff
@@ -6,4 +6,9 @@ export const ArraySelectionProperties: PropertiesTableProps = {
     type: 'Intl.ListFormatOptions',
     status: 'optional',
   },
+  variant: {
+    doc: 'Defines if the the value should be displayed in list format, or reuglar text format on one line. Defaults to `text`',
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Value/ArraySelection/ArraySelectionDocs.ts:10]:
```suggestion
    doc: 'Defines if the value should be displayed in list format, or reuglar text format on one line. Defaults to `text`',
```

File: packages/dnb-eufemia/src/extensions/forms/Value/ArraySelection/ArraySelection.tsx [hunk 2]
```diff
@@ -7,40 +7,84 @@ import { LOCALE } from '../../../../shared/defaults'
...
+import { Li, Ol, Ul } from '../../../../elements'
 
 export type Props = ValueProps<Array<number | string>> & {
   /**
    * Formatting options for the value.
    * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat
    */
   format?: Intl.ListFormatOptions
+  /**
+   * Defines if the the value should be displayed in list format or reuglar text format on one line.
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Value/ArraySelection/ArraySelection.tsx:19]:
```suggestion
   * Defines if the value should be displayed in list format or reuglar text format on one line.
```

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx [hunk 3]
```diff
@@ -30,6 +30,14 @@ import * as Examples from './Examples'
...
+### List variants
+
+<Examples.ListVariants />
+
+### List types
+
+<Examples.ListTypes />
+
 {/* This is not yet supported */}
 {/* ### Field.ArraySelection with path */}
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx:42]: Do we have an example of the out commented code? Can you remove this if so?

**Review by tujoworker — APPROVED** [review]
Nice 👍

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx (hunk 3 above)
Comment (joakbjerk) [reply packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Value/ArraySelection/demos.mdx:42]: I'll double check 👍

**Pull request merged**